### ISSUE TITLE
Feature request: Add a "name" for specific paths

### ISSUE THREAD

**Issue by cfgnunes** [flawiddsouza/favorite-folders#1]
The problem is: for very long paths, it is impossible to see the name folder directly.
For example, I have some folders in:
```
"~/Documents/Education - Research/My papers/10 - Paper NEW/Source/"
"~/Documents/Education - Research/My papers/11 - Paper NEW/Source/"
"~/Documents/Education - Research/My papers/12 - Paper NEW/Source/"
```

The idea is to enable the user to put in config an alias or a label for this path to show only the name on the main list of favorites.

So, in the config can be (example):
```
"favoriteFolders.baseFolders": [
    {
        "name": "Paper 10",
        "path": "~/Documents/Education - Research/My papers/10 - Paper NEW/Source/"
    },
    {
        "name": "Paper 11",
        "path": "~/Documents/Education - Research/My papers/11 - Paper NEW/Source/"
    },
    {
        "name": "Paper 12",
        "path": "~/Documents/Education - Research/My papers/12 - Paper NEW/Source/"
    },
]
```

So, in "Favorite Folders" list, show only the name (if exists), such as:
```
Paper 10
Paper 11
Paper 12
```

**Comment by flawiddsouza** [comment]
Hi @cfgnunes, this is now implemented and published as v1.3.0.

Instead of
```
"favoriteFolders.baseFolders": [
    {
        "name": "Paper 10",
        "path": "~/Documents/Education - Research/My papers/10 - Paper NEW/Source/"
    },
    {
        "name": "Paper 11",
        "path": "~/Documents/Education - Research/My papers/11 - Paper NEW/Source/"
    },
    {
        "name": "Paper 12",
        "path": "~/Documents/Education - Research/My papers/12 - Paper NEW/Source/"
    },
]
```

You'll have:
```
"favoriteFolders.baseFoldersExtended": [
    {
        "name": "Paper 10",
        "path": "~/Documents/Education - Research/My papers/10 - Paper NEW/Source/"
    },
    {
        "name": "Paper 11",
        "path": "~/Documents/Education - Research/My papers/11 - Paper NEW/Source/"
    },
    {
        "name": "Paper 12",
        "path": "~/Documents/Education - Research/My papers/12 - Paper NEW/Source/"
    },
]
```

I've added `favoriteFolders.baseFoldersExtended` so we don't break the extension for existing users. Now both options are available.

**Comment by cfgnunes** [comment]
Excellent! Thank you so much! 🙌
